x-pack/plugins/cloud_security_posture/public/test/utils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { screen } from '@testing-library/react';
+
+export const toBeOrNotToBe = ({ be = [], notToBe = [] }: { be: string[]; notToBe: string[] }) => {
```

Comment:
Not in love with this abstraction, I feel it doesn't help you that much and is not as clear as using pure assertions. In addition, this only applies to test IDs and using other types of queries will need to be done without this.

If you choose to keep it, please give it a more explicit name that reflects:
1. This is an assertion (should start with the word `expect`)
2. This asserts test IDs
3. This asserts existence in the document